### PR TITLE
Setup logger in SessionConfiguration

### PR DESCRIPTION
**Pull request by PChmiel**
Setup logger in SessionConfiguration

### CODE REVIEW

File: C0T0/Classes/ApiService/ApiService.swift [hunk 1]
```diff
@@ -44,12 +46,11 @@ public class ApiService {
             urlRequestBuilder: StandardURLRequestBuilder(host:configuration.host),
             urlRequestSender: StandardURLRequestSender(urlSession: URLSession.shared),
             converter: StandardURLRequestSenderErrorConverter(),
-            configuration: ApiDataSourceConfiguration()
+            configuration: ApiDataSourceConfiguration(loggingEnabled: configuration.loggingEnabled)
```
Comment (Vyeczorny) [inline C0T0/Classes/ApiService/ApiService.swift]: We should pass a `keyDecodingStrategy` here as well, it is not used anywhere right now.

**Review by Vyeczorny — CHANGES_REQUESTED** [review]

**Review by Vyeczorny — APPROVED** [review]

File: C0T0/Classes/ApiService/ApiService.swift [hunk 2]
```diff
@@ -22,11 +22,13 @@ public struct SessionConfiguration {
     let host: String
     let keyDecodingStrategy: JSONDecoder.KeyDecodingStrategy
     let requestDecorator: RequestDecorator
+    let loggingEnabled: Bool
```
Comment (TomaszPietrowski) [inline C0T0/Classes/ApiService/ApiService.swift:25]: Side note:
AFAIK we should prefer `isLoggingEnabled` name in such case

File: C0T0/Classes/ApiService/ApiService.swift [hunk 3]
```diff
@@ -44,12 +46,11 @@ public class ApiService {
             urlRequestBuilder: StandardURLRequestBuilder(host:configuration.host),
             urlRequestSender: StandardURLRequestSender(urlSession: URLSession.shared),
             converter: StandardURLRequestSenderErrorConverter(),
-            configuration: ApiDataSourceConfiguration()
+            configuration: ApiDataSourceConfiguration(keyDecodingStrategy: configuration.keyDecodingStrategy, loggingEnabled: configuration.loggingEnabled)
```
Comment (TomaszPietrowski) [inline C0T0/Classes/ApiService/ApiService.swift:49]: No changes in `ApiDataSourceConfiguration`?

**Review by TomaszPietrowski — APPROVED** [review]

File: C0T0/Classes/ApiService/ApiService.swift (hunk 3 above)
Comment (PChmiel) [reply C0T0/Classes/ApiService/ApiService.swift:49]: Both arguments were set by default

File: C0T0/Classes/ApiService/ApiService.swift (hunk 2 above)
Comment (PChmiel) [reply C0T0/Classes/ApiService/ApiService.swift:25]: Fixed in `Support for url encoded parameters and httpBody` pr

**Pull request merged**